app/models/container_node.rb

```diff
@@ -78,4 +79,19 @@ def ipaddress
   def cockpit_url
     URI::HTTP.build(:host => ipaddress, :port => 9090)
   end
+
+  supports :common_logging do
+    unless ext_management_system.respond_to?(:common_logging_route_name)
+      unsupported_reason_add(:common_logging, _('This provider type does not support common_logging'))
+    end
+  end
+
+  def common_logging_query
+    nil # {}.to_query # TODO
+  end
+
+  def common_logging_path
+    hostname = (name || "").split('.').first
```

Comment:
@simon3z I should have explained myself better:
I changed the query below to filter for an OR between `name` and `hostname`. If `name` will be nil then `hostname` will be nil as well and their `#{name}` will return an empty string, which will cause this query to match all values and the user will be presented with the regular discovery page presenting all of the logs.
do you think I should add this as a comment?